frontend/src/metabase/app.js

```diff
@@ -14,70 +14,108 @@ import 'angular-http-auth';
 
 import "./controllers";
 import "./directives";
-import "./filters";
-import "./forms";
 import "./services";
 
-import "./icons";
 
-import "./auth/auth.module";
-import "./home/home.module";
-import "./pulse/pulse.module";
-import "./query_builder/query_builder.module"
-import "./setup/setup.module";
-import "./user/user.module";
+import React from "react";
+//import { render } from "react-dom";
```

Comment:
Yeah, that was left there from an initial attempt to completely remove angular and bootstrap the app directly using just redux.  I backed off that approach and so i just commented out a few things so it was easy to restore when the time came.